db/seeds.rb

```diff
@@ -9,11 +9,11 @@
     Employee.create!(email: "employee#{l}@test.com", password: 'password')
 end
 
-Kudo.create(title: 'From seeds', content: 'Something', giver: Employee.find_by(email: 'employee1@test.com'), receiver: Employee.find_by(email: 'employee3@test.com'))
-
-Admin.create(email: "admin@test.com", password: 'password')
-
 company_values = ['Honesty', 'Ownership', 'Accountability', 'Passion']
 company_values.each do |t|
   CompanyValue.create(title: t)
 end
+
+Kudo.create(title: 'From seeds', content: 'Something', giver: Employee.find_by(email: 'employee1@test.com'), receiver: Employee.find_by(email: 'employee3@test.com'), company_value_id: 1)
```

Comment:
Hardcoding id may not be a good idea, because it will crash if you won't have an id=1 for some random reason
```suggestion
Kudo.create(title: 'From seeds', content: 'Something', giver: Employee.find_by(email: 'employee1@test.com'), receiver: Employee.find_by(email: 'employee3@test.com'), company_value: ComapnyValue.first)
```